apps/src/blocksCommon.js

```diff
@@ -250,7 +279,8 @@ function installJoinBlock(blockly) {
 function installCommentBlock(blockly) {
   blockly.Blocks.comment = {
     init: function () {
-      Blockly.cdoUtils.setHSV(this, 0, 0, 0.6);
+      // Comment blocks use a hard-coded HSV color and are not compatible with themes.
+      Blockly.cdoUtils.handleColorAndStyle(this, [0, 0, 0.6]);
```

Comment:
Would it still be worth adding a constant for the hsv for comment and unknown blocks? That way the colors are in one place.

Reply:
Yeah, that makes sense. I left them out because they were only used once but it makes sense to keep them all in one place either way.